agent/discover/cluster.go

```diff
@@ -0,0 +1,91 @@
+package discover
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+
+	"github.com/ClusterLabs/ha_cluster_exporter/collector/pacemaker/cib"
```

Comment:
I would put a comment here to raise awareness. In the future we will need to maybe replace this by specific code

Reply:
comment added

Reply:
Or move it to some kind of library. The go-pacemaker sounds like a good candidate and this project needs some love.